### DISCUSSION TITLE
App launches with 404 on windows help

### DISCUSSION THREAD

**Discussion by FrankLaterza** [tauri-apps/tauri#8108] [Q&A]
I've been working on this project for a while and Tauri has been great. But I launched my app yesterday day and I saw some errors relating to localhost which I ignored and moved on. This morning when I tried to start working on it with cargo Tauri dev I found that the app would launch and just show the 404 page. I checked the console log and it says Failed to load resource: the server responded with a status of 404 (Not Found). It's strange because this seems to only be an issue on my Windows computer and some older versions of the project seem to work fine. I just need help solving this issue. I'm assuming it's related to my Windows localhost??? It's strange because I can see the localhost:3000 work in my browser, but the Tauri app launches as 404. This is also not an issue when I launch the built app. Also, the console log says Failed to load resource: the server responded with a status of 404 (Not Found)

**Comment by FabianLars** [comment]
Hi 👋 
Can you post the output of the `tauri info` command for a start please?

**Comment by FrankLaterza** [comment]
So I restarted the project and I found the mistake I made. This came from me changing the "identifier": "com.tauri.dev" to "identifier": "my-project-name" in the tauri.config.json. I made this change while on my Linux computer and couldn't pin down the exact change that caused the 404 error on my Windows computer. This is me documenting in case there are any other dummies.
